cernopendata/jsonschemas/records/record-v1.0.0.json

```diff
@@ -286,6 +286,71 @@
           "description": "A description of the methodology used for the production of this data/software",
           "type": "string"
         },
+        "steps": {
+          "items":[
+            {
+              "properties": {
+                "configuration_files": {
+                  "items": [
+                    {
+                      "properties": {
+                        "script": {
+                          "description": "A cmsdriver script",
+                          "type":"string"
+                        },
+                        "title": {
+                          "description": "Title of the script",
+                          "type":"string"
+                        },
+                        "cms_confdb_id": {
+                          "description": "the cms_confdb_id (applies to CMS records)",
+                          "type":"string"
+                        },
+                        "process": {
+                          "description": "Indicating the process level (e.x. HLT, RECO etc.)",
+                          "type":"string"
+                        },
+                        "recid": {
+                          "description": "Record id on open data portal ...record/id",
+                          "type":"string"
+                        },
+                        "url": {
+                          "description": "External link to the configuration file",
+                          "type":"string"
+                        }
+                      },
+                      "type":"object"
+                    }
+                  ],
+                  "type":"array"
+                },
+                "generators": {
+                  "items": [
+                    {
+                      "description": "Generator used to in producing dataset",
+                      "type":"string"
+                    }
+                  ],
+                  "type":"array"
+                },
+                "global_tag": {
+                  "description": "The global tag for generator",
+                  "type":"string"
+                },
+                "release": {
+                  "description": "CMS software used in releasing data",
+                  "type":"string"
+                },
```

Comment:
To which CMSSW version does this refer? If it is the one used in production (i.e. the one which can be found in CMSDAS or in McM), then this would be "CMS software version used in production"